docs/getting-started.rst

```diff
@@ -68,3 +67,22 @@ Installation
 
 .. _conda create: https://docs.conda.io/projects/conda/en/latest/commands/create.html?highlight=create
 .. _conda install: https://docs.conda.io/projects/conda/en/latest/commands/install.html?highlight=install
+
+.. _optional-dependencies:
+
+Optional dependencies
+---------------------
+
+``xcdat`` includes APIs that are enabled by installing optional dependencies. conda will automatically attempt to install these optional dependencies with ``xcdat``. However, some optional dependencies might be skipped if your platform does not support it (e.g., ``osx-arm64``, ``windows``). ``windows`` users can try `WSL2`_ as a workaround.
```

Comment:
conda doesn't actually seem to be installing optional dependencies (`xesmf`) if they are listed under `run_constrained` in the package recipe, so I'll probably need to edit this section.

> The other solution is at the packaging level, where esmf(py) is deemed an "optional" dependency (with soft imports, or something), using run_constrained to mark what versions are compatible in the event the user ends up installing it (potentially following the advice given by the soft-import). For example:
> -- https://github.com/xCDAT/xcdat/issues/315#issuecomment-1233905146